service/notification/NotificationProcessor.php

```diff
@@ -231,8 +242,7 @@ public function processNotification($unprocessedNotification)
                     if ($order->getCurrentState() === \Configuration::get('ADYEN_OS_WAITING_FOR_PAYMENT')) {
                         $this->orderService->updateOrderState(
                             $order,
-                            \Configuration::get('PS_OS_CANCELED'),
-                            $extraVars
+                            \Configuration::get('PS_OS_CANCELED')
                         );
```

Comment:
If I'm not mistaken it's needed here as well.

```suggestion
                        );
                        
                        $this->setPspReferenceUsingNotificationData($order, $unprocessedNotification);
```

Reply:
Suggestion committed